classes/submission/Collector.php

```diff
@@ -626,6 +638,18 @@ public function getQueryBuilder(): Builder
         // Filter out excluded submission IDs
         $q->when($this->excludeIds !== null, fn (Builder $q) => $q->whereNotIn('s.submission_id', $this->excludeIds));
 
+        $q->when(
+            $this->reviewIds !== null,
+            fn (Builder $q) => $q
+                ->whereIn(
+                    's.submission_id',
+                    fn (Builder $q) => $q
+                        ->select('ra.submission_id')
+                        ->from('review_assignments AS ra')
+                        ->whereIn('ra.review_id', $this->reviewIds)
+                )
```

Comment:
I think it's better to use `exists()` with the outer field (`s.submission_id`).

Reply:
Yeah, if the number of retrieved records is small, I don't think it's going to change much, but I think it's generally better to use the `exists()`.